### PR TITLE
Add erc20 top20

### PR DESCRIPTION
**Pull request by leohar**
[Moonbeam](https://moonbeam.moonscan.io/tokens):
USDC, USDC bridged, BUSD, MATIC, DAI, AVAX, WBTC, WBTC bridged, FRAX, FTM, FXS, WGLMR, BOBA, BEPRO, mbXEN, CONV, GLINT, EFT, WETH
[Moonriver](https://moonriver.moonscan.io/tokens):
USDT, USDC, DAI, FRAX, FXS, xcKSM, IDIA, WMOVR, CWS, ZLK, SOLAR, HYDRO, RIB, BEANS, SPACE

### CODE REVIEW

File: assets/evm/v1/assets_dev.json [hunk 1]
```diff
@@ -11,5 +11,445 @@
...
+        "name": "USD Coin",
+        "priceId": "usd-coin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/USDC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0x931715FEE2d06333043d11F658C8CE934aC61D0c"
+            }
+        ]
+    },
```
Comment (valentunn) [inline assets/evm/v1/assets_dev.json:40]: Lets leave only one USDC for now (one that is "multichain")

File: assets/evm/v1/assets_dev.json [hunk 2]
```diff
@@ -11,5 +11,445 @@
...
+        "name": "USD Coin",
+        "priceId": "usd-coin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/USDC.svg",
+        "instances": [
+            {
+                "chainId": "401a1f9dca3da46f5c4091016c8a2f26dcea05865116b286f60f668207d1474b",
+                "contractAddress": "0xE3F5a90F9cb311505cd691a46596599aA1A0AD7D"
+            }
+        ]
+    },
```
Comment (valentunn) [inline assets/evm/v1/assets_dev.json:286]: We should combine the same assets into a single element with multiple "instances"
For example, instances for USDC should contain two elements - one for Moonbeam and another for Moonriver. We should also apply same logic to other overlapping tokens

File: assets/evm/v1/assets_dev.json [hunk 3]
```diff
@@ -11,5 +11,445 @@
...
+        "precision": 8,
+        "name": "Wrapped BTC",
+        "priceId": "wrapped-bitcoin",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/tokens/white/WBTC.svg",
+        "instances": [
+            {
+                "chainId": "fe58ea77779b7abda7da4ec526d14db9b1e9cd40a217c34892af80a9b332b76d",
+                "contractAddress": "0xE57eBd2d67B462E9926e04a8e33f01cD0D64346D"
+            }
+        ]
```
Comment (valentunn) [inline assets/evm/v1/assets_dev.json:117]: Same - lets leave a single WBTC for one chain

**Review by valentunn — CHANGES_REQUESTED** [review]

File: assets/evm/v1/assets_dev.json (hunk 1 above)
Comment (leohar) [reply assets/evm/v1/assets_dev.json]: left multchain USDC

File: assets/evm/v1/assets_dev.json (hunk 2 above)
Comment (leohar) [reply assets/evm/v1/assets_dev.json]: combined instances of same token

File: assets/evm/v1/assets_dev.json (hunk 3 above)
Comment (leohar) [reply assets/evm/v1/assets_dev.json]: done

**Review by valentunn — APPROVED** [review]

**Pull request merged**